docker-bake.hcl

```diff
@@ -1,11 +1,11 @@
 target "alpine-base" {
-  dockerfile = "tools/docker/Dockerfile.build"
+  dockerfile = "tools/docker/build.Dockerfile"
   tags = ["ghcr.io/datadog/libdatadog-build:alpine-base"]
   target = "alpine_builder"
 }
 
 target "alpine-build" {
-  dockerfile = "tools/docker/Dockerfile.build"
+  dockerfile = "tools/docker/build.Dockerfile"
```

Comment:
Wait, are we missing a file from this PR? I initially thought this file may just be for testing, but I see it referenced here 🤨

Reply:
Re-added it, sorry for the confusion